### PR TITLE
Startup scripts and doco

### PR DESCRIPTION
**Pull request by jonbwalker**
* added commands to run build and bundle for the react app 
* added prettierignore
* removed port proxy - not needed
* updated readme

### CODE REVIEW

File: client/.prettierrc.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "es5"
-}
+  "trailingComma": "es5",
+  "printWidth": 120
+}
```
Comment (denys-davydov) [inline client/.prettierrc.json]: The unknown symbol at the line end, possibly should be removed

File: README.md [hunk 2]
```diff
@@ -22,3 +22,33 @@ Things you may want to cover:
...
+
+If this is your first time running the app:
+
+```yarn```
+
+To start the react app only:
+
+```yarn start```
+
+To start rails + the react app:
```
Comment (denys-davydov) [inline README.md]: Maybe we should add "bundle install" as a precondition to start rails server

File: client/.prettierrc.json (hunk 1 above)
Comment (jonbwalker) [reply client/.prettierrc.json]: Done! 80cee31

File: README.md (hunk 2 above)
Comment (jonbwalker) [reply README.md]: I did as we discussed. eef5f5e

**Pull request merged**